homework-12/ToDoList/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- <link rel="stylesheet" href="todo.css"> -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.9.4/css/bulma.min.css">
+    <title>To do list</title>
+</head>
+
+<body>
+    <div class="container is-three-quarters">
+        <h1 class="title">My To Do List</h1>
+        <div class="block">
+            <div class="field is-grouped box">
+                <input class="input control is-expanded is-normal" type="text" placeholder="Add Task" id="taskTitle">

```

Comment:
It's nice to have a label for the input, even if the label will be hidden to keep your code agreed with a11y standarts